dll/win32/newdev/wizard.c

```diff
@@ -25,32 +25,8 @@
 #include <shlobj.h>
 #include <shlwapi.h>
 
-HANDLE hThread;
 
-static VOID
-CenterWindow(
-    IN HWND hWnd)
-{
-    HWND hWndParent;
-    RECT rcParent;
-    RECT rcWindow;
-
-    hWndParent = GetParent(hWnd);
-    if (hWndParent == NULL)
-        hWndParent = GetDesktopWindow();
-
-    GetWindowRect(hWndParent, &rcParent);
-    GetWindowRect(hWnd, &rcWindow);
-
-    SetWindowPos(
-        hWnd,
-        HWND_TOP,
-        ((rcParent.right - rcParent.left) - (rcWindow.right - rcWindow.left)) / 2,
-        ((rcParent.bottom - rcParent.top) - (rcWindow.bottom - rcWindow.top)) / 2,
-        0,
-        0,
-        SWP_NOSIZE);
-}
```

Comment:
This function was introduced by @christophvw back in 2005 in commit 775e2dbd070abe62b04c244db1ee82bc4d4aac77 (r19070). Maybe it was needed as a hack to make the window centered.

Reply:
The point is that windows never appear centered, unless being told so (with such code, for example). The question is whether we want this; does Windows also show the wizard always centered?

Reply:
Also here I think that the function should be more clever.

https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/msconfig_new/utils.c;hb=9a523c81356bb0349b7e545611bcad8ca3325b69#l161

https://social.msdn.microsoft.com/Forums/vstudio/en-US/54f7f131-5d15-4b2d-863b-bef329e89a32/how-to-display-the-window-center-win32-api?forum=vcgeneral

https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rosapps/applications/explorer-old/utility/utility.cpp;hb=9a523c81356bb0349b7e545611bcad8ca3325b69#l48

https://www.codeproject.com/Tips/250672/CenterWindow-in-WIN32